R/learner_randomForestSRC_classif_imbalanced_rfsrc.R

```diff
@@ -16,7 +16,28 @@
 #' `r format_bib("obrien2019imbrfsrc", "chen2004imbrf")`
 #'
 #' @template seealso_learner
-#' @template example
+#' @examples
```

Comment:
example needs to be escaped if the packages it not available

Reply:
nice catch, I updated all examples from the learners I changed